tools/src/bin/gen-protos.rs

```diff
@@ -1,18 +1,17 @@
 fn main() {
     let opencensus_dir = {
         let manifest_dir = std::path::PathBuf::from(std::env!("CARGO_MANIFEST_DIR"));
-        manifest_dir.parent().unwrap().join("opencensus-proto")
+        manifest_dir.parent().unwrap().join("opentelemetry-proto")
```

Comment:
Does this mean that we no longer support generating opencensus protos? Don't we need to support both?

Reply:
I assumed we wouldn't have to update these again since OpenCensus is deprecated and the protocol effectively frozen, but I'm realizing if we update tonic in the meantime we'd need to regenerate these. I've re-added generating the OpenCensus protos.